bittensor/__init__.py

```diff
@@ -31,12 +31,24 @@
 
 
 __version__ = '0.0.0'
-__network_dim__ = 256
+__network_dim__ = 512
```

Comment:
should we make this network dim a param that we can adjust?

Reply:
it needs to be fixed or else the peers won't agree on the encoding